pkg/jx/cmd/upgrade_cli.go

```
@@ -83,3 +94,33 @@ func (o *UpgradeCLIOptions) Run() error {
 		return o.InstallJx(true, newVersion.String())
 	}
 }
+
+func (o *UpgradeCLIOptions) latestAvailableJxVersion() (*semver.Version, error) {
+	var newVersion *semver.Version
+	if runtime.GOOS == "darwin" && !o.NoBrew {
+		brewInfo, err := o.GetCommandOutput("", "brew", "info", "--json", "jx")
+		if err != nil {
+			return nil, err
+		}
+		v, err := o.latestJxBrewVersion(brewInfo)
```

It seems that this error is not handled.